### ISSUE TITLE
When you change a key you kill recursion

### ISSUE THREAD

**Issue by glucena** [fiznool/express-mongo-sanitize#2]
Next example explains the problem: 

```
> var mongoSanitize = require('express-mongo-sanitize')
> var obj = {'k.1':{'k.2': 1}};
> mongoSanitize.sanitize(obj, {replaceWith:'_'})
{ k_1: { 'k.2': 1 } }
```

As you can see, keys under first level doesn't change.

**Comment by fiznool** [comment]
Thanks for the bug report. Do you think you'd be able to put together a PR which fixes this?

**Comment by pedrovagner** [comment]
If some day I need to use this feature, I promise that I'll create a PR. BTW, nice package.

**Comment by fiznool** [comment]
Fixed and published as v1.3.1 to npm.
